src/core/inspect.js

```diff
@@ -37,7 +38,7 @@ function inspect(x) {
     return ` "${x}"`
   }
 
-  if(isSymbol(x)) {
+  if(isSymbol(x) || isDate(x)) {
```

Comment:
Curious why we do this?
In the browser should output the string version of the date already.
Is there a case you know of where this is not true?

Reply:
So doing no conversion does returns something like this, "2018-12-23T01:04:17.707Z" whereas calling `toString` returns this, "Sat Dec 22 2018 17:07:07 GMT-0800 (Pacific Standard Time)" which I thought was a nicer version to see for inspect, what do you think?